components/observations/ObservationsListView.tsx

```diff
@@ -50,22 +54,55 @@ export const ObservationsListView: React.FunctionComponent<ObservationsListViewP
   const mapResult = useMapLayer(center_id);
   const mapLayer = mapResult.data;
 
-  const nacObservationsResult = useNACObservations(center_id, requestedTime);
-  const nacObservations: ObservationFragment[] = nacObservationsResult.data?.pages?.flatMap(page => page.data) ?? [];
-  const nwacObservationsResult = useNWACObservations(center_id, requestedTime);
-  const nwacObservations: ObservationFragment[] = nwacObservationsResult.data?.pages?.flatMap(page => page.data) ?? [];
-  const observations: ObservationFragment[] = nacObservations
+  // TODO brian wire up duration correctly - there's an interaction between useInfiniteQuery (which has a cached
+  // notion of whether there's more data) and this duration, and I'm not sure of the best way to solve it.
+  // Right now, the queryKey in useNACObservations/useNWACObservations doesn't use the duration, so it's not invalidated when the duration changes.
+  // That has the nice property of not re-fetching data when the duration changes, but it also means that if you make the duration longer, you won't get new data.
+  const nacObservationsResult = useNACObservations(center_id, requestedTime, DEFAULT_OBSERVATIONS_WINDOW);
```

Comment:
@stevekuznetsov this is the bit that I'm not sure how best to solve.

maybe one approach would be to make `DEFAULT_OBSERVATIONS_WINDOW` a constant that's as big a lookback as we'd ever allow, and then change the `fetchMoreData` logic in this component to look at the date of the oldest observation that's been fetched and compare it to the current date filter (instead of just looking at `hasNextPage` the way it does today).

Reply:
I'm going to try and implement that approach as a followup PR (along with any other feedback)